strings/README.md

```diff
@@ -24,6 +24,7 @@
 2. [All subsequences](java/sequence.java)
 3. [KMP String Searching](java/kmp.cpp)
 4. [Rabin Karp String Searching](java/rabin-karp.cpp)
+4. [String Tokenizer](java/tokenizer.java)
```

Comment:
Hi @toihirhalim Please convert `4` to `5` because there is already a program with number 4

Reply:
Hi @TawfikYasser  
I'm changing it now